### DISCUSSION TITLE
No Ticket Found Text

### DISCUSSION THREAD

**Discussion by winsanmwtv** [Limaru-Minecraft/iciwi#53] [Ideas]
Instead of error like this

java.lang.NullPointerException: Cannot invoke "java.util.TreeMap.size()" because "fareClasses" is null

Make it show text like "No tickets found from your selected route." when you select from CustomTicket machine and orgin-destination can't find a correct route to go when buying paper ticket.

**Comment by Mineshafter61** [comment]
very late to the party but i think i finished this
